nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,59 @@
+const express = require("express");
+const app = express();
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+//Let's again create a calculator. This time the calculator will support 4 features(+ - / *)
+//All the numbers that should be inputted should be gotten from the query parameters: calculator?firstParam=1&secondParam=2...
+
+app.get("/add", async (req, res) => {
+  const params = req.query;
+  const values = Object.values(params).flat();
+  const add = values.reduce(
```

Comment:
Consider parsing the params to either Int or Float before making additions/subtractions etc..

Reply:
Hi thanks for reviewing my homework's🙏🏻